suites/pacific/rbd/tier-2_rbd_regression.yaml

```diff
@@ -144,6 +144,12 @@ tests:
       name: Test snap and clone creation on restored image from trash
       polarion-id: CEPH-11413
 
+  - test:
+      desc: Verify parent-clone relationship and clone IOs are not affected when parent moved to trash
+      module: rbd_clone_trash_restore_parent_image.py
```

Comment:
Module and name looks conflicting for clone trash restore or parent trash restore. can we correct it meaningful.

Reply:
Done. Please let me know if this new name looks okay.